docs/dockstore-introduction.rst

```diff
@@ -1,138 +1,46 @@
-register tools that point at locations like Docker Hub, Seven Bridges,
-and Amazon ECR. You can substitute WDL and Nextflow for the descriptor
-language for workflows. You might even be able to mix and match
-descriptor languages eventually!
-
-Over time, we find "skinny" Docker images, those with single tools
-installed in them, are more helpful for extending and building new
-workflows with. That being said, "fat" Docker containers, which include
-multiple tools and even full workflows with frameworks like
-`SeqWare <https://seqware.github.io/>`__ or
-`Galaxy <https://galaxyproject.org/>`__, can have their place as well.
-Projects like the ICGC `PanCancer Analysis of Whole
-Genomes <https://dcc.icgc.org/pcawg>`__ (PCAWG) made use of "fat" Docker
-containers that had complex workflows that fully encapsulated alignment
-and variant calling. The self-contained nature of these Docker
-containers allowed for mobility between a wide variety of environments
-and greatly simplified the setup of these pipelines across a wide
-variety of HPC and cloud environments. Either approach works for the
-Dockstore so long as you can describe the tool or workflow inside the
-Docker container as a CWL/WDL-defined tool (which you can for most
-things).
+The Dockstore concept is simple: provide a place where users can share tools encapsulated in Docker and described with the workflow languages. This enables scientists to share analytical tools in a way that makes them machine readable and runnable in a variety of environments. While Dockstore is focused on serving researchers in the biosciences, the combination of Docker and workflow languages can be used by anyone to describe the tools and services in their `Docker images <https://docs.docker.com/get-started/overview/#docker-objects>`__ in a standardized, machine-readable way. 
+
+By providing an environment where scientific tools can easily be created, maintained, browsed, and launched, Dockstore aligns itself with several trends supporting open science. But it is more than just an open ecosystem to facilitate open science -- we're involved in shaping what open science looks like. Dockstore works closely with the `GA4GH <https://www.ga4gh.org>`__, and accordingly, Dockstore supports the two languages most commonly used by GA4GH's `Cloud Work Stream <http://ga4gh.cloud/>`__ members and APIs: `Common Workflow Language <https://www.commonwl.org/>`__ (CWL) and `Workflow Description Language <https://openwdl.org/>`__ (WDL).  Dockstore also attempts to work with new and alternative languages/standards such as `Nextflow <https://www.nextflow.io/>`__ as popular challengers to CWL and WDL. We also work on the `GA4GH Tool Registry <https://github.com/ga4gh/tool-registry-service-schemas>`__ standard as a way of sharing data with workflow platforms and partners, in addition to working on developing task execution, workflow execution, and data transfer standards at the GA4GH. 
```

Comment:
Not a fan of the "the two languages most commonly used by GA4GH's `Cloud Work Stream" language. 
First, I'm not sure its true. Second, it's not very politic. 

I suggest throwing in Nextflow and Galaxy and then revising to simply say, something like "four of the most prominent languages" for example.

Reply:
Yeah, I was trying to preserve the older emphasis on CWL and WDL (which we technically still kind of have) without radically changing the older wording, but on second reading it is a bit sketch. Will reword it.

Reply:
That's good to hear, because when I wrote it I didn't like it either, but was mislead by the earlier form into thinking I had to align GA4GH with only WDL and CWL (in the same sense that NIH would prefer we not stick their logo next to an unassociated company's logo). If that's not the case then this is an easy fix. Thanks for the clarification.